server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java

```diff
@@ -230,9 +231,12 @@ public Invocations getInvocations() {
                if (ignoredCaches.contains(cacheName)) {
                   cacheInfo.status = "IGNORED";
                } else {
-                  // This action will fail for ignored caches
-                  Cache cache = restCacheManager.getCache(cacheName, request);
-                  cacheInfo.status = cache.getStatus().toString();
+                  if(cacheHealth != HealthStatus.FAILED) {
+                     Cache cache = restCacheManager.getCache(cacheName, request);
+                     cacheInfo.status = cache.getStatus().toString();
+                  } else {
+                     cacheInfo.status = "FAILED";
```

Comment:
use HealthStatus enum

Reply:
```cacheInfo.status``` is not the health of the cache, but as far as I could see, it is the component status or "ignored", so using the HealthEnum here would be confusing

Reply:
And I can't use ```ComponentStatus.FAILED``` because when a cache fails to start, it is basically inaccessible  :/

Reply:
Used ComponentStatus.FAILED